### PR TITLE
 London Class 8 - Mahri Atdayeva- JS-core 3 - Week 1

### PR DESCRIPTION
**Pull request by mahri09**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: mandatory/debugging-practice/script.js [hunk 1]
```diff
@@ -89,13 +89,14 @@ function render() {
...
 
     //add delete button to every row and render again
-    let delButton = document.createElement("button");
+    let delBut = document.createElement("button");
     delBut.id = i + 5;
     cell5.appendChild(delBut);
     delBut.className = "btn btn-warning";
     delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
+    delBut.addEventListener("click", () => {
```
Comment (HamraKhan) [inline mandatory/debugging-practice/script.js:97]: Looks good to me Mahri :) I believe there are more bugs to be fixed in this task aside from the ones you have done.

File: mandatory/thinking-like-a-programmer/task.md [hunk 2]
```diff
@@ -29,12 +29,18 @@ After you've watched these videos I'd like you to answer these questions
...
+-first thing I am doing relaxing just relax don't panic,
+-after try to read code step by step, console log it to see what is happening on each step,
 
 ## 3. What should you do if you get stuck?
 
-<!-- Write your answer here -->
+-and then nothing could find google is your best friend, trying to find same errors as mine, stackoverflow.com is the my favorite site which I go first,or anything else
+-if you have friend or colleague who might help with your problem then ask them
+-if haven't solve your problem then make some coffee or tea or just give a rest, or if it is evening time sleep well and take a look tomorrow with fresh mind it will look much more different then now
+-but if don't have time and you have lots of works then do another work and try problem solving after!
```
Comment (HamraKhan) [inline mandatory/thinking-like-a-programmer/task.md:46]: Really nice you added other concepts aside from the ones that were in the links. :)

Comment (HamraKhan) [inline mandatory/thinking-like-a-programmer/task.md:46]: Really nice you added other concepts aside from the ones that were in the links. :)

File: mandatory/debugging-practice/script.js (hunk 1 above)
Comment (mahri09) [reply mandatory/debugging-practice/script.js:97]: I will take a look again and also review it, I checked it quickly if you have any options so please share!

Comment (mahri09) [reply mandatory/debugging-practice/script.js:97]: thank you!

File: mandatory/thinking-like-a-programmer/task.md (hunk 2 above)
Comment (mahri09) [reply mandatory/thinking-like-a-programmer/task.md:46]: Thanks a lot Hamra!

**Pull request closed without merging**